test/functional/rpc_net.py

```diff
@@ -249,11 +249,17 @@ def test_addnode_getaddednodeinfo(self):
         added_nodes = self.nodes[0].getaddednodeinfo()
         assert_equal(len(added_nodes), 1)
         assert_equal(added_nodes[0]['addednode'], ip_port)
+        # check that filtering by node works
+        self.nodes[0].addnode(node="11.22.33.44", command='add')
+        added_node = self.nodes[0].getaddednodeinfo(node=ip_port)
+        assert_equal(added_nodes, added_node)
```

Comment:
I'm not sure if we test for any list of added_nodes greater than a length of 1

Reply:
Yes, I can add this verification.

Reply:
Done.